pkg/library/libraryinputresources/prune_mustgather.go

```diff
@@ -52,6 +53,40 @@ func GetRequiredInputResourcesFromMustGather(ctx context.Context, inputResources
 		return nil, err
 	}
 
+	inputDirResources, err := LenientResourcesFromDirRecursive(mustGatherDir)
```

Comment:
This is probably the wrong function. I think we want to be able to prune and separately we want to be able to check. I suspect there different use-cases